docs/releases/1.9.6.txt

```diff
@@ -12,3 +12,5 @@ Bugfixes
 * Added support for relative path redirects to
   ``SimpleTestCase.assertRedirects()`` because Django 1.9 no longer converts
   redirects to absolute URIs (:ticket:`26428`).
+
+* Prevented data loss of microseconds on MySQL and SQLite (:ticket:`26498`).
```

Comment:
Please be a bit more specific -- I believe this issue affects `TimeField`, right?